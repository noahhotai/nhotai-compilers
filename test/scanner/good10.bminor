==
!=
!=
sdfsdf

sdfsdf
;
sdfasdfdsaf;

+
+
++
chel;
chelswea;