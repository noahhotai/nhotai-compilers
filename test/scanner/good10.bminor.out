EQUALITY       ==
IEQUALITY     !=
IEQUALITY     !=
IDENTIFIER     sdfsdf
IDENTIFIER     sdfsdf
SEMICOLON      ;
IDENTIFIER     sdfasdfdsaf
SEMICOLON      ;
ADD            +
ADD            +
INCREMENT      ++
IDENTIFIER     chel
SEMICOLON      ;
IDENTIFIER     chelswea
SEMICOLON      ;
